manga109api/manga109api.py

```diff
@@ -17,6 +19,66 @@ def __init__(self, root_dir):
         with (self.root_dir / "books.txt").open("rt", encoding='utf-8') as f:
             self.books = [line.rstrip() for line in f]
 
+
+    def get_annotation_ordered(self, book, annotation_type="annotations"):
+        """
+        Given a book title, return the annotation in the form of a dict.
+
+        In the raw XML in the Manga109 dataset, the bounding box data in the
+        `page` tag is not sorted by its annotation type, and each bounding
+        box type appears in an arbitrary order. The output dictionary of this
+        function preserves the ordering of each bounding box tag in the raw
+        XML data, mainly for data editing purposes. Note that the ordering of
+        the bounding box tags does not carry any useful information about the
+        contents of the data.
+
+        Caution: Due to the aforementioned feature, the format of the output
+        dictionary differs from the `get_annotation` function.
+
+        Args:
+            book (str): A title of a book. Should be in self.books.
+
+        Returns:
+            annotation (dict): Annotation data consists of dict.
+        """
+        assert book in self.books
+
+        def keys_renamed(d):
+            for k in list(d.keys()):
+                try:
+                    if str(d[k]) == str(int(d[k])):
```

Comment:
Which key needs this if function? 
Could you add a simple comment (e.g. `# ~~~ key needs this`) here?

Reply:
Thanks, in the new code, I added a clarification of the keys that are affected:
https://github.com/manga109/manga109api/blob/add-ordered-output-func/manga109api/manga109api.py#L48